llvm/utils/TableGen/SearchableTableEmitter.cpp

```diff
@@ -55,6 +69,8 @@ struct GenericEnum {
   const Record *Class = nullptr;
   std::string PreprocessorGuard;
   MapVector<const Record *, Entry> Entries;
+  bool Scoped = false;
+  unsigned ScopedSize = 0;
```

Comment:
nit: rename it to SizeInBits? It seems this has nothing to do with it being a scoped enum vs regular enum.